CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Changelog
 
+## v6.0.0-beta.8 - 2017-10-05
+
+### Breaking changes
+
+* Changed `New-Service` to return error when given unsupported `-StartupType` and fixed `Set-Service` icon failing test. (#4802)
+* Allow `*` to be used in registry path for `Remove-Item`. (#4866)
+* Remove unsupported `-ShowWindow` switch from `Get-Help`. (#4903)
+* Fix incorrect position of a parameter which resulted in the args passed as input instead of as args for `InvokeScript()`. (#4963)
+
+### Engine updates and fixes
+
+* Make calls to `void CodeMethod` work. (#4850) (Thanks @powercode)
+* Get `PSVersion` and `GitCommitId` from the `ProductVersion` attribute of assembly (#4863) (Thanks @iSazonov)
+* Fix `powershell -version` and built-in help for `powershell.exe` to align with other native tools. (#4958 & #4931) (Thanks @iSazonov)
+* Load assemblies with `Assembly.LoadFrom` before `Assembly.Load` when the file path is given. (#4196)
+* Add a generic file watcher function in `HelpersCommon.psm1`. (#4775)
+* Update old links and fix broken links in `docs/host-powershell/README.md`. (#4877)
+* Fix when importing remote modules using version filters (and added tests). (#4900)
+* Enable transcription of native commands on non-Windows platforms. (#4871)
+* Add a new line to `CommandNotFoundException` error string. (#4934 & #4991)
+* Fix bug where PowerShell would exit with an error within an SSH remoting connection on Linux. (#4993)
+
+### General cmdlet updates and fixes
+
+* Added `Remove-Service` to Management module. (#4858) (Thanks @joandrsn)
+* Added functionality to set credentials on `Set-Service` command. (#4844) (Thanks @joandrsn)
+* `Select-String` changes:
+    * Exclude directories (as opposed to individual files) discovered from `-Path`. (#4829) (Thanks @iSazonov)
+    * Support more argument completion scenarios by adding `ArgumentCompletionsAttribute`. (#4835) (Thanks @iSazonov)
```

Comment:
#4835 was to add argument completion support to `Get-Date`, not `Select-String`.